lesson_01/AngelicaCastillo/readme.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="deco.css">
+</head>
+<body>
+   <h2>Angelica Castillo</h2>
+   <hr>
+    <ul>
+        <li>Angelica is the daughter of immmigrant parents. She is 1 of 6 children. She has worked in retail and has expirience working in Early Childhood Education. Taught at Wilmington Head Start, Bush Early Education, & Mount Pleasant Elementary. Angelica is persuing a software development carrear at Code Differently.</li>
+    </ul>
+    <h3> Get to know me</h3>
+    <p>Angelica is the daughter of immmigrant parents. She is 1 of 6 children. She has worked in retail and has expirience working in Early Childhood Education. Taught at Wilmington Head Start, Bush Early Education, & Mount Pleasant Elementary. Angelica is persuing a software development carrear at Code Differently. </p>
```

Comment:
This paragraph is different than the one in your original markdown readme. Please adjust.